.github/workflows/test-all.yml

```diff
@@ -1,11 +1,7 @@
 name: Run All Tests
 
 on: 
-  pull_request:
-    branches:
-      - master
-    types:
-      - closed
+  push:
   schedule:
     - cron: '0 6,18 * * *'
```

Comment:
Do we still need the cron job?

Reply:
I guess it hasn't been very helpful. Removing.